build_runner/README.md

```diff
@@ -99,14 +100,71 @@ directories are served, and on what ports.
 For example to serve the `example` and `web` directories on ports 8000 and 8001
 you would do `dart run build_runner serve example:8000 web:8001`.
 
-##### test
+### test
 
 The test command will forward any arguments after an empty `--` arg to the
 `dart run test` command.
 
 For example if you wanted to pass `-p chrome` you would do
 `dart run build_runner test -- -p chrome`.
 
+### build-filter
```

Comment:
We should link this section from the general `Command Line Options` heading, adding an entry to the list with all the other options.